machida/lib/wallaroo/experimental/__init__.py

```diff
@@ -93,10 +93,10 @@ def connect(self, host=None, port=None):
                 else:
                     raise
 
-    def write(self, message, event_time=0):
+    def write(self, message, key=None, event_time=0):
```

Comment:
I think `key=None` should follow `event_time=0` in this parameter list, to avoid breaking any existing code that worked with the assumption there were only two parameters and the second was `event_time`, e.g.
```my_connector.write(msg, 123)``` would break (or produce an unintelligible string when you try to convert the byte contents of `123` into a string the `received` function above in Pony)